examples/softkiller/README.md

```diff
@@ -0,0 +1,24 @@
+# JetReconstruction.jl Examples - SoftKiller 
+
+## `softkiller_plots.jl`
+
+This example can be run as:
+
+```
+julia --project=examples examples/softkiller_plots.jl  --maxevents=100 --grid-size=0.4  --algorithm=Kt/ --pileup-file=test/data/sk_example_PU.hepmc --hard-file=test/data/sk_example_HS.hepmc 
```

Comment:
Nice! However, can I request that the example be given assuming that this is the user's current working directory (it's the model we use in all of the other cases). So it should be more like...

```sh
julia --project softkiller_plots.jl  --maxevents=100 --grid-size=0.4  --algorithm=Kt --pileup-file=../../test/data/sk_example_PU.hepmc.zst --hard-file=test/data/sk_example_HS.hepmc.zst
```

- The name of the algorithm is also fixed (no trailing slash)
- The input files should be Zstd compressed

There is also a dependency issue here - the `examples` project does not have the correct dependencies for the plot example. So you need to generate a separate `Project.toml` file in this directory with all of the dependencies for these examples.